cdc/http_handler.go

```diff
@@ -48,6 +50,12 @@ type commonResp struct {
 	Message string `json:"message"`
 }
 
+// ChangefeedCommonInfo holds some common usage information of a changefeed
+type ChangefeedCommonInfo struct {
+	ID      string         `json:"id"`
+	Summary ChangefeedResp `json:"summary"`
```

Comment:
Probably a separate struct dedicated to our public RESTful API is better.

Reply:
done